### ISSUE TITLE
running in local, 'max retries exceeded' to token endpoint

### ISSUE THREAD

**Issue by kakarlus** [openid-certification/oidctest#170]
same issue described by #52 but that thread didn't share a solution

**Comment by zandbelt** [comment]
This indicates a local connectivity problem ("Connection refused") for which there is no generic solution. If you're running the Docker instances, you should make sure that your instances can find/connect to each other using Docker networking.

**Comment by kakarlus** [comment]
Hello @zandbelt i can actually test the auth endpoint but not the token endpoint.

**Comment by zandbelt** [comment]
yes, the authz endpoint is accessed from the browser, the token endpoint between containers

**Comment by kakarlus** [comment]
Hi @zandbelt, does it matter when my OP is not using Docker?
